api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java

```diff
@@ -1248,7 +1248,7 @@ public DataSetDefinition constructIntensiveMonitoringDataSet() {
             "startDate=${revisionEndDate-2m+1d},revisionEndDate=${revisionEndDate-1m},location=${location}"),
         "");
 
-    CohortIndicator MI15DEN16 =
+    /*CohortIndicator MI15DEN16 =
```

Comment:
@ningosi any reason to comment this out?

Reply:
Yah, MI CAT15 will have a ticket done. that section was blocking smooth testing of the other categories. You can help by maybe uncomment and build it, then try running the report, but that will be handled in the upcoming changes for CAT 15 MI

Reply:
I will revert that back